### ISSUE TITLE
Layout: Incorrect spacing

### ISSUE THREAD

**Issue by glacierbits** [trezor/trezor-suite#2328]
As discussed: White box <> Left account menu (or Dashboard white box <> App edges) are too "crowded" and should have more space in between. See [design for 1440](https://app.zeplin.io/project/5ee87c00f6af719a645d14c3/screen/5ee8c6343e7221b6cdc90705). 

### Example width: 1440
<img width="1440" alt="Screen Shot 2020-09-09 at 19 50 40" src="https://user-images.githubusercontent.com/29627086/92635228-bfea9880-f2d5-11ea-853b-099c8767064d.png">

### Example width: 1050
<img width="1050" alt="Screen Shot 2020-09-09 at 19 51 58" src="https://user-images.githubusercontent.com/29627086/92635330-ed374680-f2d5-11ea-86ab-856446446416.png">

**Comment by slowbackspace** [comment]
duplicate of https://github.com/trezor/trezor-suite/issues/2139, but this one has nice description so I am closing the old issue :)

tldr; To match the designs we should implement different max-widths for content in dashboard and wallet/accounts.

**Comment by bosomt** [comment]
**QA OK**

suite dev master, web commit fb5994a71e797046bfdf18048a763d98ba8653ff
device TT 2.3.3 commit 0d5f00668
NIXOS + Chromium 85.0.4183.83
